### DISCUSSION TITLE
How can we see traces of MULTIMODAL llms over langfuse ?

### DISCUSSION THREAD

**Discussion by Aaryaveerkrishna23** [langfuse/langfuse#2510] [Ideas]
### Describe the feature or potential improvement

In image to image generation how is langfuse helpful ?

**Comment by marcklingen** [comment]
Multi-modal traces are on the near-term roadmap post [v3](https://github.com/orgs/langfuse/discussions/1902#discussioncomment-9729809). Currently you can log out traces with references to media assets to langfuse as links are clickable but this is currently limited. Thanks for your plus one on this.

**Comment by johnwalz97** [comment]
We are also looking to trace multi-modal (base64 images for now) prompts with LangFuse. Looking forward to when this is released.

**Reply by itssimon** [reply]
This is exactly our use case as well.

**Comment by marliessophie** [comment]
We've [just added support for images](https://langfuse.com/changelog/2024-08-21-openai-multi-modal-traces) (via url), and are planning support for base64 images after the v3 release. We'd also love to hear if other modalities would be of interest to you, please comment on [this thread](https://github.com/orgs/langfuse/discussions/3004) if you have feedback or would like to upvote. Looking forward to hear feedback!
